modopt/opt/proximity.py

```diff
@@ -1042,3 +1042,69 @@ def _cost_method(self, *args, **kwargs):
             print(' - K-SUPPORT NORM (X):', cost_val)
 
         return cost_val
+
+
+class GroupLASSO(ProximityParent):
+    """This class implements the proximity operator of the group-lasso
+    regularization, with groups dimension being the first dimension
+    Attributes:
```

Comment:
Please try to keep the doc strings consistent. This should be `Parameters` and should have a space separating it from the class description.